site/components/src/main/java/uk/nhs/digital/website/beans/ApiCatalogue.java

```diff
@@ -0,0 +1,17 @@
+package uk.nhs.digital.website.beans;
+
+import org.hippoecm.hst.content.beans.Node;
+import org.onehippo.cms7.essentials.dashboard.annotations.HippoEssentialsGenerated;
+
+@Node(jcrType = "website:apicatalogue")
+@HippoEssentialsGenerated(internalName = "website:apicatalogue", allowModifications = false)
+public class ApiCatalogue extends CommonFieldsBean {
```

Comment:
We definitely don't need the current getter methods because they have no meaning in context of API Catalogue document.

I know that this leaves the class empty but am not sure if we can remove it completely, or would that break the rendering?

Reply:
i assume we need the class to be here (even if empty) since `dynamic_bean_generation` is set to false

once we have introduce the taxonomy filter, we'll probably need a method here to return the listcomponent of internal links so i'll just leave it for now?

Reply:
Indeed, good point. No harm leaving it in. At the very least, it gives the bean a more precise name.